docs/admin.rst

```diff
@@ -370,6 +405,288 @@ Nginx rewrite rules allow you to simulate the familiar URL convention. The follo
 
   }
 
+.. _pgrst_errors:
+
+PostgREST Error Codes
+=====================
+
+PostgREST error codes have the form ``PGRSTgxx``, where ``PGRST`` is the prefix that differentiates the error from a PostgreSQL error, ``g`` is the group where the error belongs and ``xx`` is the number that identifies the error in the group.
+
+.. _pgrst0**:
+
+Group 0 - Connection
+--------------------
+
+Related to the connection with the database.
+
++---------------+---------------------------------------------+
+| Code          | Description                                 |
++===============+=============================================+
+| .. _pgrstx01: | Could not connect with the database due to  |
+|               | an incorrect :ref:`db-uri` or due to the    |
+| PGRST001      | PostgreSQL service not running.             |
++---------------+---------------------------------------------+
+| .. _pgrstx02: | Could not connect with the database due to  |
+|               | an internal error.                          |
+| PGRST002      |                                             |
++---------------+---------------------------------------------+
+| .. _pgrstx03: | Could not connect with the database when    |
+|               | building the :ref:`schema_cache` due to the |
+| PGRST003      | PostgreSQL service not running.             |
++---------------+---------------------------------------------+
```

Comment:
Added this as an alternative to using subheadings. Opinions on which one would be better to use are welcome.

Reply:
@laurenceisla Yeah, this looks much better!

Seems the anchor link is not clickable though, any way to work around that?

Even if that doesn't work I'd still favor it over subheadings.

---

Found a discussion about anchor links in table rows https://github.com/STEllAR-GROUP/hpx/issues/4495. Seems it's possible to do them.

Reply:
> Found a discussion about anchor links in table rows https://github.com/STEllAR-GROUP/hpx/issues/4495. Seems it's possible to do them.

That's good, although it keeps the anchor link visible at all times, not at hover only. I'll leave that solution (the hover anchor links) for another PR.